### PR TITLE
feat(additional-support-for-the-elderly): new application

### PR DESCRIPTION
**Pull request by ylfahfa**
# Application for Additional Support for the Elderly

## Why

for applicants to apply for additional support for the elderly at Social Insurance Administration (TR)

/umsoknir/felagslegur-vidbotarstudningur/

## Gervimenn that should work for this application

Sayed Wahab ÞÍ Sverrisson - 0101479
Thelma Hrönn ÞÍ Sveinsdóttir - 0101469
Helgi Fannar ÞÍ Merzaye - 0811479
Anna Mary ÞÍ Ívarsdóttir - 2011489
Rögnvaldur ÞÍ Gunnarsson - 3111439
Natanael ÞÍ Einarsson - 1801429
María ÞÍ Lulic - 0111439
Helgi Fannar ÞÍ Jónsson - 1701469
Kolbrún Helga ÞÍ Jarrah - 0111489
Ljósbrá ÞÍ Torp - 0611419
Elmar ÞÍ Merzaye - 1801419

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by karenbjorg — APPROVED** [review]

File: libs/feature-flags/src/lib/features.ts [hunk 1]
```diff
@@ -25,6 +25,7 @@ export enum Features {
   complaintsToAlthingiOmbudsman = 'isComplaintToAlthingiOmbudsmanEnabled',
   healthcareLicenseCertificate = 'isHealthcareLicenseCertificateEnabled',
   transferOfMachineOwnership = 'isTransferOfMachineOwnershipEnabled',
+  additionalSupportForTheElderlyApplication = 'isAdditionalSupportForTheElderlyEnable',
```
Comment (saevarma) [inline libs/feature-flags/src/lib/features.ts:28]: ~ Up to you if you want to add the 'd' at the end 😅
```suggestion
  additionalSupportForTheElderlyApplication = 'isAdditionalSupportForTheElderlyEnabled',
```

**Review by saevarma — APPROVED** [review]
The 2 core files LGTM

**Review by Thorrik58 — COMMENTED** [review]
Looks good just some minor comments.

File: libs/application/templates/social-insurance-administration/additional-support-for-the-elderly/src/forms/AdditionalSupportForTheElderlyForm.ts [hunk 2]
```diff
@@ -0,0 +1,523 @@
...
+          .receivedAwaitingIncomePlanTitle,
+      alertMessage:
+        socialInsuranceAdministrationMessage.conclusionScreen
+          .incomePlanAlertMessage,
+      alertType: 'warning',
+      expandableDescription:
+        additionalSupportForTheElderyFormMessage.conclusionScreen.bulletList,
+      expandableIntro:
+        additionalSupportForTheElderyFormMessage.conclusionScreen.nextStepsText,
+      bottomButtonLink: 'https://minarsidur.tr.is/forsendur/tekjuaetlun',
```
[the remark this answers is not in the document]
Comment (karenbjorg) [reply libs/application/templates/social-insurance-administration/additional-support-for-the-elderly/src/forms/AdditionalSupportForTheElderlyForm.ts:500]: expected type for buttomButtonLink is string not MessageDescriptor

> (property) bottomButtonLink?: string | undefined

[the remark this answers is not in the document]
Comment (Thorrik58) [reply libs/application/templates/social-insurance-administration/additional-support-for-the-elderly/src/forms/AdditionalSupportForTheElderlyForm.ts:500]: We should probably address that 😅

**Review by Thorrik58 — APPROVED** [review]

**Pull request merged**